css/form_subscribe.css

```diff
@@ -15,6 +15,7 @@
   position: absolute;
```

Comment:
We really love what you've done here. The change is specific enough. Good job team!